scripts/check_for_inconsistent_pandas_namespace.py

```diff
@@ -7,52 +7,97 @@
 This is meant to be run as a pre-commit hook - to run it manually, you can do:
 
     pre-commit run inconsistent-namespace-usage --all-files
+
+To automatically fixup a given file, you can pass `--replace`, e.g.
```

Comment:
Is there a preference for how to fix the file if both usages occur?

Reply:
currently I'm just removing the `pd.` prefix if both occur